src/ctapipe/image/muon/ring_fitter.py

```diff
@@ -32,22 +51,44 @@ class MuonRingFitter(Component):
 
     fit_method = traits.CaselessStrEnum(
         list(FIT_METHOD_BY_NAME.keys()),
-        default_value=list(FIT_METHOD_BY_NAME.keys())[0],
+        default_value="kundu_chaudhuri_taubin",
     ).tag(config=True)
 
-    def __call__(self, fov_lon, fov_lat, img, mask):
-        """allows any fit to be called in form of
-        MuonRingFitter(fit_method = "name of the fit")
+    def __call__(self, geom, image, clean_mask):
```

Comment:
Might as well also put in the type hints here like I showed in my example. It's not required, but makes using it in an editor with code completion and type checking much nicer.